README.md

```diff
@@ -1 +1,51 @@
-# IdolGAN
\ No newline at end of file
+# IdolGAN ~~[adoːkẽꜜɴ]~~
+
+We use [HiFaceGAN](https://github.com/Lotayou/Face-Renovation) and [KID-F Dataset](https://github.com/PCEO-AI-CLUB/KID-F).
+
+# Usage
+### Environment
+- Ubuntu/CentOS
+- PyTorch 1.0+
+- CUDA 10.1
+- python packages: opencv-python, tqdm, 
+- Data augmentation tool: [imgaug](https://imgaug.readthedocs.io/en/latest/source/installation.html#installation-in-pip)
+- [Face Recognition Toolkit](https://github.com/ageitgey/face_recognition) for evaluation
+- [tqdm](https://github.com/tqdm/tqdm) to make you less anxious when testing:)
+
+### Checkpoint
+We provide Model Checkpoint which trained from KID-F Dataset. We used train_dataset of KID-F for training, which takes 2 days on a GTX 1080 Ti. We upload the checkpoint on our [googledrive](https://drive.google.com/drive/folders/1GrZIofQc3uWFVWserxgPEO97DZEJYOuK?usp=sharing).
+
+### Configurations
+The configurations is stored in `options/config_hifacegan.py`, the options should be self-explanatory, but feel free to leave an issue anytime.
```

Comment:
Maybe we can explain how to edit config_hifacegan.py for our KID-F dataset

Reply:
Hey DongGyu it's your Job. Thx :) @vkehfdl1